SpatialGDK/Source/SpatialGDKEditorToolbar/Private/SpatialGDKEditorToolbar.cpp

```diff
@@ -126,14 +126,32 @@ void FSpatialGDKEditorToolbarModule::StartupModule()
 		}
 	});
 
-	LocalDeploymentManager->Init(GetOptionalExposedRuntimeIP());
+	LocalDeploymentManager->Init();
 	LocalReceptionistProxyServerManager->Init(GetDefault<USpatialGDKEditorSettings>()->LocalReceptionistPort);
 
 	SpatialGDKEditorInstance = FModuleManager::GetModuleChecked<FSpatialGDKEditorModule>("SpatialGDKEditor").GetSpatialGDKEditorInstance();
 
 	// Get notified of map changed events to update worker boundaries in the editor
 	FLevelEditorModule& LevelEditorModule = FModuleManager::LoadModuleChecked<FLevelEditorModule>("LevelEditor");
 	FDelegateHandle OnMapChangedHandle = LevelEditorModule.OnMapChanged().AddRaw(this, &FSpatialGDKEditorToolbarModule::MapChanged);
+
+	// Grab the runtime and inspector binaries ahead of time so they are ready when the user wants them.
+	const FString RuntimeVersion = SpatialGDKEditorSettings->GetSelectedRuntimeVariantVersion().GetVersionForLocal();
+	const FString InspectorVersion = SpatialGDKEditorSettings->GetInspectorVersion();
+
+	AsyncTask(ENamedThreads::AnyBackgroundThreadNormalTask, [this, RuntimeVersion, InspectorVersion] {
```

Comment:
Should probably use weak this here

Reply:
Not sure if this works for this class type though

Reply:
Looks like other methods use this to ensure a valid ptr

if (FSpatialGDKEditorToolbarModule* Module = FModuleManager::GetModulePtr<FSpatialGDKEditorToolbarModule>("SpatialGDKEditorToolbar"))

Reply:
Hmmm, not sure if we should start doing this. We've used `this` extensively around this class. The only time we haven't is when we're calling using the public notification helpers that can be called from outside this class.

Reply:
I don't think here it's a major issue but capturing this in AsyncTask's makes me nervous because there isn't a guarantee the task runs before the module is destroyed. The task graph is shut down after the modules so it's possibly a job could be waiting, albeit a race condition! 

Although since this is not used elsewhere, maybe it's not a problem/discussion for this PR